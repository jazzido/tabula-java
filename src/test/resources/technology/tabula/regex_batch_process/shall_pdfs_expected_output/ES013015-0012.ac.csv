GASTON CO ENVIRONMENTAL HEALTH,,BOBBY FRANKLIN
991 WEST HUDSON BLVD,,
"",,231 KINGS DR
"GASTONIA, NC 28052 Courier # 06-34-01",,"KINGS MOUNTAIN, NC 28086"
EIN: 566000300EH,,
StarLiMS ID: ES013015-0012001 Date Collected:,01/27/15,Time Collected: 4:30 PM
Date Received:,01/30/15,Collected By: P Long
Sample Type: Sampling Point:,Outside spigot,Well Permit #:
Analyte,,Result,Allowable Limit,Unit Qualifier(s)
Arsenic,,< 0.005,0.010,mg/L
Barium,,< 0.1,2.00,mg/L
Cadmium,,< 0.001,0.005,mg/L
Calcium,,7,,mg/L
Chloride,,< 5.00,250,mg/L
Chromium,,< 0.01,0.10,mg/L
Copper,,< 0.05,1.3,mg/L
Fluoride,,< 0.20,4.00,mg/L
Iron,,< 0.10,0.30,mg/L
Lead,,< 0.005,0.015,mg/L
Magnesium,,2,,mg/L
Manganese,,< 0.03,0.05,mg/L
Mercury,,< 0.0005,0.002,mg/L
pH,,6.8,,N/A
Selenium,,< 0.005,0.05,mg/L
Silver,,< 0.05,0.10,mg/L
Sodium,,< 1.0,,mg/L
Sulfate,,< 5.00,250,mg/L
Total Alkalinity,,27,,mg/L
Total Hardness,,27,,mg/L
Zinc,,< 0.05,5.00,mg/L
Report Date:,02/05/2015,,Reported By:,Arnold Hall
