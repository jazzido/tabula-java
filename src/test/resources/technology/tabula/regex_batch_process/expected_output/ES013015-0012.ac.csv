Arsenic,< 0.005,0.010,mg/L
Barium,< 0.1,2.00,mg/L
Cadmium,< 0.001,0.005,mg/L
Calcium,7,,mg/L
Chloride,< 5.00,250,mg/L
Chromium,< 0.01,0.10,mg/L
Copper,< 0.05,1.3,mg/L
Fluoride,< 0.20,4.00,mg/L
Iron,< 0.10,0.30,mg/L
Lead,< 0.005,0.015,mg/L
Magnesium,2,,mg/L
Manganese,< 0.03,0.05,mg/L
Mercury,< 0.0005,0.002,mg/L
pH,6.8,,N/A
Selenium,< 0.005,0.05,mg/L
Silver,< 0.05,0.10,mg/L
Sodium,< 1.0,,mg/L
Sulfate,< 5.00,250,mg/L
Total Alkalinity,27,,mg/L
Total Hardness,27,,mg/L
Zinc,< 0.05,5.00,mg/L
